testnet/stacks-node/src/genesis.rs

```diff
@@ -0,0 +1,98 @@
+use stacks::chainstate::stacks::StacksAddress;
+use stacks::{
+    burnchains::bitcoin::address::BitcoinAddress,
+    chainstate::stacks::db::{AccountBalance, VestingSchedule},
+    util::hash::Sha256Sum,
+};
+
+pub const GENESIS_DATA_BYTES: &str = include_str!("../chainstate.txt");
+pub const GENESIS_DATA_SHA_BYTES: &str = include_str!("../chainstate.txt.sha256");
```

Comment:
Should this hash be exposed in the `v2/info` endpoint?

Reply:
Works for me

Reply:
Issue created https://github.com/blockstack/stacks-blockchain/issues/2138
Doing this in a follow up PR because right now it requires a small refactor to remove the cyclical dependency between the blockstack core lib and the lib that contains this hash.